proto/spire/api/registration/registration.proto

```diff
@@ -140,4 +186,10 @@ service Registration {
     rpc EvictAgent(EvictAgentRequest) returns (EvictAgentResponse);
     // ListAgents will list all attested nodes
     rpc ListAgents(ListAgentsRequest) returns (ListAgentsResponse);
+
+    // MintX509SVID mints an X509-SVID directly with the SPIRE server CA.
```

Comment:
Should we be considering a new GRPC service for this, it can be on the same port.

This seems like significantly different behaviour than "registration".

Reply:
Hmm, that's not a bad idea. @evan2645, thoughts?

Reply:
A new service is probably the correct thing to do, but it's also a little inconsistent because we have already polluted the "registration api" with non-registration-y things. A v2 of this API has been on our collective minds for some time now - my thinking thus far has been to let this iteration collect as many widgets along the way as we need so that we could have a clearer picture of how to carve things up when the time comes.

I haven't created an issue for this work yet because we didn't have a clear definition or vision on things like organization etc. Perhaps the time has come for us to sit down and propose something?

Anyways, with these considerations, my two cents would be on keeping this here for consistency's sake and taking this instance as another data point for the organization of the next iteration. I would not block on creating a new service now for this purpose if others here feel strongly about it, but it does seem a little cumbersome and unnatural given the current state of the registration API (and the way we are using it today).

PS>> If we _were_ to define a separate service now, I think there would be some implications in other parts of the codebase e.g. server endpoints, handlers, etc.